rules/utils/avoid-capture.js

```diff
@@ -28,10 +28,17 @@ const nameCollidesWithArgumentsSpecial = (name, scopes, isStrict) => {
 	return isStrict || scopes.some(scopeHasArgumentsSpecial);
 };
 
+const maybeGlobal = (name, scopes) => scopes.some(
```

Comment:
```suggestion
const isProbablyGlobal = (name, scopes) => scopes.some(
```

?

And maybe a doc comment that explains what maybe/probably implies. Like when can it not be correct.